ibodata/depth_profile.py

```diff
@@ -1,6 +1,36 @@
+import numpy as np
+
 from beprof.profile import Profile
 
 
 class DepthProfile(Profile):
-    """TODO"""
-    pass
+
+    def range(self, level):
```

Comment:
range will collide with python `range` keyword. Think about better name.

Reply:
I removed this method, because it's just duplication of width() from Profile